README.md

```diff
@@ -183,8 +184,18 @@ In order to install the OpenEuropa Multilingual demo module follow [the instruct
 ./vendor/bin/drush en oe_multilingual_demo -y
 ```
 
-## Contributing
+## Enabling URL suffix
+
+This is optional and should be done only if your website should follow the [IPG rules](http://ec.europa.eu/ipg/print/print_all_content/index_en.htm#3.0) for language negotiation.
 
+For enabling this feature you need to install the OpenEuropa Multilingual URL Suffix module.
```

Comment:
This line it's still there and it should be removed as @imanoleguskiza 's comment seems to have been addressed down below.